src/olympia/devhub/tests/test_views_edit.py

```diff
@@ -1472,25 +1491,38 @@ class TestEditTechnicalUnlisted(BaseTestEdit):
     __test__ = True
     listed = False
 
-    def test_whiteboard(self):
-        edit_url = self.get_url('technical', edit=True)
+    def setUp(self):
+        super().setUp()
+        self.technical_url = self.get_url('technical')
+        self.technical_edit_url = self.get_url('technical', edit=True)
 
+    def test_whiteboard(self):
         # It's okay to post empty whiteboard instructions.
-        response = self.client.post(edit_url, {'whiteboard-public': ''})
+        response = self.client.post(self.technical_edit_url, {'whiteboard-public': ''})
         assert response.context['form'].errors == {}
 
         # Let's update it.
-        response = self.client.post(edit_url, {'whiteboard-public': 'important stuff'})
+        response = self.client.post(
+            self.technical_edit_url, {'whiteboard-public': 'important stuff'}
+        )
         assert response.context['form'].errors == {}
         addon = self.get_addon()
         assert addon.whiteboard.public == 'important stuff'
 
         # And clear it again.
-        response = self.client.post(edit_url, {'whiteboard-public': ''})
+        response = self.client.post(self.technical_edit_url, {'whiteboard-public': ''})
         assert response.context['form'].errors == {}
         addon = self.get_addon()
         assert addon.whiteboard.public == ''
 
+    def test_site_permission(self):
+        self.addon.update(type=amo.ADDON_SITE_PERMISSION)
+        response = self.client.get(self.technical_url)
+        assert response.status_code == 200
+
+        response = self.client.post(self.technical_edit_url, {})
+        assert response.status_code == 403
```

Comment:
I guess the whiteboard is redundant when there aren't any reviews, but it's not really altering metadata about the site permission add-on so it'd be fine (expected) that whiteboard would still be available.

Reply:
It's a bit annoying to allow though, cause it goes through the generic `addons_section` view. I don't think it's worth it for the moment.